source/community/release_notes/sumac/dev_op_release_notes.rst

```diff
@@ -40,13 +102,28 @@ Other Operator Changes
 Deprecations & Removals
 ***********************
 
+- The existing "Content Libraries" feature has been renamed to "Legacy Libraries" and will be deprecated in the next release (Teak), and removed entirely in Ulmo in favor of the new "Content Libraries" feature. The Teak release will include a tool for migrating content from Legacy Libraries into new Content Libraries.
+- In `frontend-app-learner-dashboard <https://github.com/openedx/frontend-app-learner-dashboard>`_ 
+   - support for Optimizely has been removed along with the ProductRecommendations widget.
+   - `DEPR: Optimizely Support <https://github.com/openedx/frontend-app-learner-dashboard/issues/387>`_
+   - Removed the RecommendationsPanel widget
+   - `DEPR: RecommendationsPanel <https://github.com/openedx/frontend-app-learner-dashboard/issues/410>`_
+- In edx-platform:
+   - ``commerce-coordinator`` related code has been removed
+   - `PR: <https://github.com/openedx/edx-platform/pull/35527>`_
+- [UPCOMING] In Teak pre-design-tokens brand packages will no longer be supported. With design tokens, theme authors will instead override core Paragon tokens by defining their own JSON tokens that get deep merged alongside the core Paragon tokens, thus overriding any tokens that were defined by the theme author. See `the associated DEPR ticket for details <https://github.com/openedx/brand-openedx/issues/23>`_.
```

Comment:
another upcoming

```
[UPCOMING] Between now and the release of Teak, all Dockerfiles will be removed from the Open edX
organization. As an alternative, Tutor provides `production-ready Docker images`
<https://hub.docker.com/u/overhangio/>_ for all supported Open edX services. And each Open edX service
repository should contain documentation describing how it can be installed and executed, allowing anyone to write
a Dockerfile that provisions the repository. See `the associated DEPR ticket for details
<https://github.com/openedx/public-engineering/issues/263>`_. 
```